Unix/tests/http/test_http.cpp

```diff
@@ -185,14 +185,16 @@ static void* MI_CALL _http_client_proc(void* param)
     // read response
     char r_buf[1024];
     size_t read = 0;
+    int err;
 
     do
     {
         r = Sock_Read(sock, r_buf, sizeof(r_buf), &read);
+        err = Sock_GetLastError();
     }
-    while (r != MI_RESULT_OK && Sock_GetLastError() == 11 /*EAGAIN*/);
+    while (r != MI_RESULT_OK && (err == 11 || err == 35) /*EAGAIN and EDEADLK*/);
```

Comment:
That doesn't make sense, especially for the lower numbered (standard) error codes.

Can you please point out two systems that conflict with one another so I can take a look, thanks.